src/components/o-commercial-hero-cover/_template.js

```diff
@@ -0,0 +1,26 @@
+import html from 'nanohtml';
+
+export default ({
+                  src,
+                  alt = 'Commercial Hero cover',
+                  gradient = 'white',
+                  contentAlign = 'left'
+}, childrenFragment) => html`
+  <header class="o-commercial-hero-cover__header ${gradient === "white" ? "gradient-white" : "gradient-black"}">
+    <picture class="m-picture o-commercial-hero-cover__picture">
```

Comment:
as we dont have a molecule in the plib and we need it for AEM, please use a property for setting classes from outiside.

Reply:
a prop like "classes" for example

Reply:
or picture-classes might be better

Reply:
done